### PR TITLE
SegResNet algo add logging

### PR DESCRIPTION
**Pull request by myron**
- adds logging via logger.debug(), instead of print()
- adds tqdm for progress

### CODE REVIEW

File: auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py [hunk 1]
```diff
@@ -90,13 +90,18 @@
 from monai.metrics import DiceHelper
 
 
+from monai.apps.utils import DEFAULT_FMT
+from monai.apps.auto3dseg.auto_runner import logger
+print = logger.debug
```
Comment (mingxin-zheng) [inline auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py:95]: Will there be any info printed out if the level is set to "info"?

File: auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py [hunk 2]
```diff
@@ -355,7 +347,7 @@ def get_crop_transforms(self):
 
             num_steps_per_image = self.crop_params.get("num_steps_per_image", 1)
             if num_steps_per_image > 1:
-                print('Cropping with num_steps_per_image', num_steps_per_image)
+                print(f"Cropping with num_steps_per_image {num_steps_per_image}")
```
Comment (mingxin-zheng) [inline auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py:350]: What's the `num_steps_per_image` for?

File: auto3dseg/algorithm_templates/segresnet/configs/hyper_parameters.yaml [hunk 3]
```diff
@@ -8,15 +8,14 @@ output_classes: null
 class_names: null
 class_index: null
 
+degug: false
```
[the remark this answers is not in the document]
Comment (myron) [reply auto3dseg/algorithm_templates/segresnet/configs/hyper_parameters.yaml:11]: yes, a typo, thank you.

File: auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py (hunk 1 above)
Comment (myron) [reply auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py:95]: currently, we only use logger.debug inside of the algo, so nothing is printed in info

File: auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py (hunk 2 above)
Comment (myron) [reply auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py:350]: it's number of crops to do from the same image (used by RandCrop... transforms)

**Pull request merged**